src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/QueueOrderSchemaImportActivity/lookups.json

```diff
@@ -683,6 +683,16 @@
         },
         "sortOrder": 0,
```

Comment:
do we need to fix the sort order for reject shipping?

Reply:
"Reject" transitions are for the drawer in the order list, the sort order has no impact whatsoever on that feature.